Z-Wave: add list of used association groups (#34700)
home-assistant/home-assistant.github.io: source/_integrations/zwave_js.markdown

### Including / excluding a controller in an existing network using [classic inclusion](#classic-inclusion-versus-smartstart)

A Z-Wave controller that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave controller to join another network, because Home Assistant acts as the central hub.

## Z-Wave association groups used

> An _association_ in Z-Wave terminology is when two or more Z-Wave products communicate directly with one another without the need to communicate via a hub. An _association group_ in Z-Wave terminology is a group of devices that are controlled from a controller device. Z-Wave has a range of association groups that each has different capabilities.

Reviewer: ```suggestion
An _association_ in Z-Wave terminology is when two or more Z-Wave products communicate directly. This enables devices to communicate with each other without the need to communicate via a hub, or to send unsolicited reports to the central hub.

An _association group_ in Z-Wave terminology is a group of devices that another one will send commands to in certain situations. Association groups and their functionality are specific to the device that sends the commands. Refer to the device manual for details.
```

> In Home Assistant, only one association group is used. It is a group of 1 one device.

Reviewer: ```suggestion
Home Assistant, a single association group is supported:
```

- **Group 1**: This is an association group that includes only one device. It is used to send a **Device Reset Locally Notification** after a factory reset.

> This association group is used when Home Assistant [resets the Z-Wave controller](#controller) or a Z-Wave end device.

Reviewer: ```suggestion
This association group is used when Home Assistant [resets the Z-Wave controller](#controller).
```

Reviewer: And we should add:
Under normal circumstances, it is not necessary to add a device to this group.

A Z-Wave controller that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave controller to join another network, because Home Assistant acts as the central hub.

## Z-Wave association groups used

> Home Assistant, a single [association group](#association-group) is supported:

Reviewer: ```suggestion
In Home Assistant, a single [association group](#association-group) is supported:
```

A Z-Wave controller that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave controller to join another network, because Home Assistant acts as the central hub.

> ## Z-Wave association groups used

Reviewer: ```suggestion
## Available Z-Wave association groups
```
maybe? "Used" seems not correct, as we don't use them.
However that could imply they are actually available to users, although they need another controller with a UI to configure them.

Author: <https://github.com/home-assistant/home-assistant.io/pull/34718>
